.github/workflows/ci.yml

```diff
@@ -40,8 +40,8 @@ jobs:
       fail-fast: false
       matrix:
         # https://www.cockroachlabs.com/docs/releases/release-support-policy
-        crdb: [v22.2, v23.1, v23.2]
-        ruby: [head]
+        crdb: [v23.2, v24.1, v24.2]
+        ruby: ["3.3"]
```

Comment:
Ruby 3.3 is the LTS branch, and the current ruby-head on github is a beta version...